docs/introduction.rst

```diff
@@ -1,26 +1,26 @@
 An Introduction to ``sunraster``
 ================================
 
-``sunraster`` is a free, open-source, SunPy-affiliated package that provides
-tools to manipulate and visualize slit spectrograph data using the Python
+``Sunraster`` is a free, open-source, SunPy-affiliated package that provides
+the tools to manipulate and visualize slit spectrograph data using the Python
```

Comment:
```suggestion
tools to manipulate and visualize slit spectrograph data using the Python
```

It doesn't supply all the tools you could want for this, which is what I take away from the use of ```the tools```.